addons/website_blog/tests/common.py

```diff
@@ -35,3 +36,10 @@ def setUp(self):
             'notification_type': 'inbox',
             'groups_id': [(6, 0, [group_public_id])]
         })
+        self.user_portal = Users.with_context({'no_reset_password': True}).create({
```

Comment:
If only one test require it, create it either in the test class either in the test directly.